pkg/controller/hyperconverged/hyperconverged_controller_test.go

```diff
@@ -96,6 +96,7 @@ var _ = Describe("HyperconvergedController", func() {
 			})
 
 			It("should ignore invalid requests", func() {
+				hcoNamespace := commonTestUtils.NewHcoNamespace()
```

Comment:
This is done in many tests, couldn't this namespace be created only once and be reused in all of them?

Reply:
done

Reply:
in `pkg/controller/operands/operandHandler_test.go` there's the same issue

Reply:
done